### DISCUSSION TITLE
Can I allow users in any OIDC group to login with OIDC?

### DISCUSSION THREAD

**Discussion by poperigby** [mealie-recipes/mealie#3867] [Q&A] [answered]
I don't want to have to set `OIDC_USER_GROUP` and then add every single user to a group.

**Comment by cmintey** [comment]
Yes, that environment variable is optional. If you leave it empty, then any OIDC user should be able to log in to Mealie.

**Reply by poperigby** [reply]
I must be encountering a bug then, because that isn't working if I also have `OIDC_ADMIN_GROUP` set to something, but `OIDC_USER_GROUP` unset. I can login with a user who's part of the `OIDC_ADMIN_GROUP`, but no other users.

**Reply by cmintey** [reply]
Could be a bug, I'll take a closer look later today

**Reply by poperigby** [reply]
Thanks :slightly_smiling_face:

**Reply by cmintey** [reply]
Does the user you are trying to log in with and fail have any groups?

**Reply by cmintey** [reply, marked as answer]
I believe this is the issue: https://github.com/mealie-recipes/mealie/issues/3871

**Reply by poperigby** [reply]
Great, thanks for getting it fixed so quick!
